.github/workflows/publish.yml

```
@@ -0,0 +1,23 @@
+name: Publish
+
+on:
+  push:
+    tags:
+      - v0.**
```

We can perform this action for all versions

```suggestion
      - v*
```